earth_osm/utils.py

```diff
@@ -131,7 +131,34 @@ def convert_pd_to_gdf_lines(df_way):
     gdf.drop(columns=["lonlat"], inplace=True)
     return gdf
 
-def output_csv_geojson(df_feature, primary_name, feature_name, region_list, data_dir):
+
+def write_csv(df_feature, outputfile_partial, feature_name, out_aggregate, fn_name):
+    """Create csv file. Optimized for large files as write on disk in chunks"""
+    if out_aggregate:
+        output_path = os.path.join(outputfile_partial, f"_{feature_name}s" + ".csv")
+        df_feature.to_csv(
+            output_path, index=False, header= not os.exists(output_path), mode="a"
```

Comment:
this will just pick the header from the first dataframe... requires changes to the way we take column names